### PR TITLE
Change copper blood from ferrous to metallic

### PR DESCRIPTION
**Pull request by Daxxi3**
## About the PR

Changed the description of copper blood from being ferrous to metallic.

## Why / Balance

Bothered me that spider blood was called ferrous despite using copper instead.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
metallic blood

**Changelog**

:cl:
-Changed copperblood from ferrous to metallic when examined

### CODE REVIEW

**Comment by JoeTheJanitor** [comment]
‘Cuprous’ might be a more consistent naming scheme, given that normal blood is described as ‘ferrous’.

**Comment by Daxxi3** [comment]
> ‘Cuprous’ might be a more consistent naming scheme, given that normal blood is described as ‘ferrous’.
Yeah I looked for an equivalent but couldn't find that word.

**Comment by JoeTheJanitor** [comment]
> Yeah I looked for an equivalent but couldn't find that word.

(look to bottom for TL;DR)

Well, I looked up ‘cuprous’ and found this:
Cuprous
adjective (chemistry)
of copper with a valence of one; of copper(I).

but the fact that it refers to copper of a specific valence interested me, because I think chemistry is cool. So, simply put, from some quick googling I found that hemocyanin (the equivalent of hemoglobin in blue blood which contains copper instead of iron) alternates between an oxidized and unoxidized state; when it is not oxidized, it is colorless, and when it is oxidized, it is blue. When the blood is oxidized, as it must be in-game because it always appears blue, the valence of the copper in the hemocyanin would actually be TWO, not one. From doing a little more googling:
Cupric
adjective (chemistry)
of copper with a valence of two; of copper(II).

Of course, that raises the question of wether normal blood should actually be referred to as ‘ferric’ instead of ‘ferrous’, but that’s a discussion for another time!

so, TL;DR: I believe that the most scientifically accurate description would actually be ‘cupric’. Thank you for coming to my TED talk.

**Comment by chromiumboy** [comment]
Thank you for your contribution. I think metallic is a fine description for blue blood. If anyone wants to change the description to 'cupric' or similar they are welcome to open a new PR
